src/components/projects-summary/projects-summary.html

```diff
@@ -72,15 +69,24 @@ <h4 class="catalog-modal-title">
       </div>
     </div>
   </div>
-  <h2 class="secondary">Resources</h2>
-  <span>
-    {{$ctrl.resourcesText}}
-  </span>
-  <div class="resources-panel">
-    <ul>
-      <li ng-repeat="resource in $ctrl.resources">
-        {{resource.text}}
-      </li>
-    </ul>
+  <div ng-if="$ctrl.showGetStarted">
+    <div class="getting-started-panel">
+      <h2 class="secondary" ng-if="$ctrl.projects && $ctrl.projects.length > 0">Getting Started</h2>
+      <button class="getting-started-button btn btn-default" ng-class="{'with-projects': $ctrl.projects && $ctrl.projects.length > 0}" ng-click="$ctrl.handleGettingStartedClick()">
+        <span class="fa fa-info-circle fa-2"></span>
+        <span class="getting-started-button-text">Start Guided Tour</span>
+      </button>
+    </div>
+    <div class="resources-panel">
+      <h2 class="secondary">Resources</h2>
+      <p>
+        {{$ctrl.resourceDescription}}
+      </p>
+      <ul>
+        <li ng-repeat="resource in $ctrl.resourceLinks">
+          <a href="" ng-click="$ctrl.onResourceClick(resource)">{{resource.title}}</a>
```

Comment:
I've updated it for the resource links here. Adding this for project link would require a bit of rework since the href is not known here. We could pass in a base url rather than a callback if this is desired. Thoughts?